modules/rntuple.mjs

```diff
@@ -205,6 +201,7 @@ fieldListIsList = fieldListSize < 0;
     if (flags & 0x4) checksum = reader.readU32();
 
      fieldDescriptors.push({
+        fieldRecordSize,
```

Comment:
This doesn't need to be saved in the FieldDescriptor (it's not a property of the Field, but rather of its parent frame)